### ISSUE TITLE
bottomTabNavigator: tabPress -> no popToTop

### ISSUE THREAD

**Issue by kaspolar** [IjzerenHein/react-navigation-shared-element#81]
I nested my _SharedElementStackNavigator_ in a _BottomTabNavigator_. If I click on the corresponding Icon in the _BottomTabNavigator_ the _TabNavigator_ navigates to the _SharedElementStackNavigator_ (if it isn't already open), but the _popToTop_ action of the _SharedElementStackNavigator_ isn't fired. If I replace the _createSharedElementStackNavigator_ with the normal _createStackNavigator_ everything works fine, but, of course, the Animation is lost. 

So I have the problem, that I don't get navigated to the initialRoute of the _SharedElementStackNavigator_, if I press on the corresponding Icon in the TabBar, although that's normally standard.

Im am using the v5-navigation.

How do I solve this issue? Or is a fix planned?

**Comment by darrylyoung** [comment]
I also just came across this issue.

Did you find a workaround to keep the old functionality but have the new animation, @jakobmeusburger? Thanks.

**Comment by jakobmeusburger** [comment]
> I also just came across this issue.
> 
> Did you find a workaround to keep the old functionality but have the new animation, @jakobmeusburger? Thanks.

No, not until now. I am using it without the old functionality for now. I really hope that this gets fixed.

**Comment by tedi** [comment]
@jakobmeusburger @darrylyoung I ran into the same thing. I am working around it by adding a `tabPress` listener and checking the current index. If I detect that the user is already in that stack and not on the first index, I then `popToTop`.

Hope that helps.
